### PR TITLE
API provides functionality for user to add products to current order

### PR DESCRIPTION
**Pull request by PaulineBA**
### User Story
```As a restaurant API
In order for users to buy food they want from the restaurant
I would like them to be able to add multiple products to an order for a specific user
```
## Changes proposed in this pull request:
Editing the Orders Controller, the routes and the spec file.

## What I have learned working on this feature:
* We learned how to create Update action in the controller.
* We need to always declare a method in the test before calling on them

### CODE REVIEW

File: app/controllers/api/v1/orders_controller.rb [hunk 1]
```diff
@@ -4,4 +4,12 @@ def create
     order.order_items.create(product_id: params[:product_id])
     render json: {message: 'The product has been added to your order', order_id: order.id}
   end
+
+  def update
+    order = Order.find(params[:id])
+    product = Product.find(params[:product_id])
+    order.order_items.create(product: product)
+    render json: {message: 'The product has been added to your order', order_id: order.id}
+end
```
Comment (jysmys) [inline app/controllers/api/v1/orders_controller.rb]: tab it in one step...

**Review by jysmys — APPROVED** [review]
Looks good to me, just one small intention!

**Review by erikbjoern — COMMENTED** [review]
The PR and request spec should be called order_controller_spec.

**Review by erikbjoern — CHANGES_REQUESTED** [review]
The request spec should be called orders_controller_spec or something with create / update I think.

**Review by erikbjoern — APPROVED** [review]

**Pull request merged**